pkg/k8shandler/deployment.go

```diff
@@ -113,6 +113,8 @@ func (node *deploymentNode) create() error {
 		if err != nil {
 			if !errors.IsAlreadyExists(err) {
 				return fmt.Errorf("Could not create node resource: %v", err)
+			} else {
+				return node.pause()
```

Comment:
so if it already exists, you are pausing it?

Reply:
the function is idempotent. we are ensuring that it is paused, yes.